packages/runtime/src/state/react-builder-preview.ts

```diff
@@ -685,11 +703,24 @@ function makeswiftApiClientSyncMiddleware(
   }
 }
 
+type BufferedMessages = [Action, Transferable[]?][]
+
 class MessageChannel {
   private channel: MessagePort | null = null
-  private bufferedMessages: [Action, Transferable[]?][] = []
+  private bufferedMessages: BufferedMessages = []
+  private bufferedMessagesAfterInitialized: BufferedMessages = []
+  private initialized = false
```

Comment:
Not suggesting that we change anything here, but there is some tension here between "initialization" and "setup": are they the same, if not, why not?